src/watchkit.cs

```diff
@@ -1637,9 +1672,13 @@ interface WKExtendedRuntimeSession {
 	[BaseType (typeof (WKInterfaceObject), Name = "WKInterfaceAuthorizationAppleIDButton")]
 	[DisableDefaultCtor] // Handle is `nil`
 	interface WKInterfaceAuthorizationAppleIdButton {
-
 		[Export ("initWithTarget:action:")]
+		[Deprecated (PlatformName.WatchOS, 6,1, message: "Use initWithStyle:target:action:")]
```

Comment:
I think we have to use the C# method names here instead of the ObjC selectors

Reply:
Yup, not sure what the convention is for referring to a c# constructor that's replacing the existing constructor. cc: @spouliot

Reply:
I think we used `.ctor(...)` in some places but
`Use "new WKInterfaceAuthorizationAppleIdButton (WKInterfaceVolumeControlOrigin,NSObject,Selector)' instead."`
might be better

in doubt use `grep` on the `/src/` directory ;-)

Reply:
> Yup, that's a a placeholder,

Put a `do-not-merge` label if there are pending questions on your side. That way someone else won't merge it (e.g. if you're not online). You can remove the label once you have answers.